pkg/hoist/failing_scripts_test_hoist_launchable/installs/testLaunchable_abc123/bin/launch

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+exit 0
```

Comment:
just so I'm clear... I'm guessing this now needs to be added because the `failing_scripts_test_hoist_launchable` test would otherwise have a missing `bin/launch` cahsing an error. let me know if I'm wrong.